### ISSUE TITLE
List, edit and save automatic reply messages 

### ISSUE THREAD

**Issue by alexanderhoset** [IFRCGo/cbs#358]
Ref cbs/Design/cbs.pdf ("Feedback messages admin", Add Feedback message admin, Edit Feedback message 1 admin, Edit Feedback message 2 admin )  // to be added to the design file

If you want to start before we have the mockup done for admin its basically the same as in  cbs/Design/cbs.pdf ("Feedback messages", Add Feedback message, Edit Feedback message 1, Edit Feedback message 2 ) 

Note: the theme flatly is used today, this should be changed to custom bootstrap variables. Ref #353

**Comment by alexanderhoset** [comment]
This dependent on #303 .

**Comment by karolikl** [comment]
Duplicate of #275
